### ISSUE TITLE
How Do I Use NiceHash

### ISSUE THREAD

**Issue by BlitzedOfficial** [UnamSanctam/SilentCryptoMiner#1791]
Hello There

How Do I Use NiceHash With This Miner

Thank You

**Comment by UnamSanctam** [comment]
You can just connect to the NiceHash pools normally, and for the XMR miner you just enable "NiceHash Mining".

**Comment by BlitzedOfficial** [comment]
Would I use the stratum generator?

**Comment by BlitzedOfficial** [comment]
And instead of my wallet adress I should put the miner adress?

**Comment by UnamSanctam** [comment]
Yes you can use the stratum generator, and you can either put the wallet address of your NiceHash account or an external BTC wallet if you want that.

**Comment by BlitzedOfficial** [comment]
what the best wallet for Raven coin

**Comment by UnamSanctam** [comment]
There are many good ones, you can find many different wallets for Ravencoin here: https://ravencoin.org/wallet/. I guess ZelCore, Exodus and Atomic Wallet are pretty common for it now.

**Comment by BlitzedOfficial** [comment]
Thank you 

and one last question

when in use of the stratum mechanism, In The URL Do I Include The Word Stratum At The Start Or Only Include The Present URL

**Comment by UnamSanctam** [comment]
You only include the pool address and port, don't include the stratum part. So this for example: `kawpow.auto.nicehash.com:443`

**Comment by BlitzedOfficial** [comment]
Thank You Sir

All My Questions Have Been Answered

Very Good Support

**Comment by BlitzedOfficial** [comment]
Just A Quick One

I Have An Unsurity

Im Using NiceHash To Mine Raven And The It Says It Will Pay Me In BTC. 

Is This Normal, I Want It To Give Me RVN Coin. 

I Have Never Used NiceHash Before So Im Unsure

![image](https://github.com/UnamSanctam/SilentCryptoMiner/assets/141662956/0d8f378e-06a8-48de-87cb-8c7f70c7592d)

**Comment by UnamSanctam** [comment]
Yes NiceHash is a hashrate marketplace and not a standard pool, so people are purchasing your hashrate and paying you in BTC. If you want RVN then you should mine on a normal pool, for example 2miners.

**Comment by BlitzedOfficial** [comment]
what is the difference between 
![image](https://github.com/UnamSanctam/SilentCryptoMiner/assets/141662956/8fece6c0-ec10-4255-8f78-b7975ddacc39)
And 
![image](https://github.com/UnamSanctam/SilentCryptoMiner/assets/141662956/92e6adc9-f8fd-4ec9-9d42-c050b7f91413)

**Comment by BlitzedOfficial** [comment]
and also what Would Happen If I Bought Hash From The Marketplace On NiceHash

**Comment by BlitzedOfficial** [comment]
Would It Increase My Speeds And Performance

**Comment by UnamSanctam** [comment]
>what is the difference between

It's the share difficulty, meaning how long it will on average take for you to find a share. You should normally always pick the lowest one, the "NiceHash" difficulty is only for those with a very large amount of hashrate (it's called a "NiceHash" difficulty because of people purchasing a large amount of hashrate on Nicehash).

>and also what Would Happen If I Bought Hash From The Marketplace On NiceHash

You would just buy the hash from other miners, but the amount of money it costs is pretty much the same as if you just buy the cryptocurrency normally on an exchange.

>Would It Increase My Speeds And Performance

No the NiceHash marketplace is for purchasing the hashrate of other people.

**Comment by BlitzedOfficial** [comment]
for the 2miners would you use stratum or https

**Comment by UnamSanctam** [comment]
Stratum for port `6060`, Stratums (SSL) for port 16060.

**Comment by BlitzedOfficial** [comment]
it wont let me surpass one rig or one worker

**Comment by BlitzedOfficial** [comment]
it only shows one current worker

**Comment by UnamSanctam** [comment]
Yes that's how pools work, they combine miners if they have the same worker name. If you want multiple workers then you need a unique worker name for each miner, for example my miner has `{COMPUTERNAME}` that you can use which will automatically replace that with the miners computer name.

**Comment by BlitzedOfficial** [comment]
could you get banned for doing that?

**Comment by UnamSanctam** [comment]
It depends, sometimes some pools have limits on the amount of workers or having a lot of workers could be suspicious. Normally it won't lead to a ban though.

**Comment by BlitzedOfficial** [comment]
nice

**Comment by BlitzedOfficial** [comment]
i think i messed it up

**Comment by BlitzedOfficial** [comment]
it was working  min ago but now its declining
![image](https://github.com/UnamSanctam/SilentCryptoMiner/assets/141662956/0db716e0-c41e-44e4-9348-f46adcf76c7e)

**Comment by BlitzedOfficial** [comment]
I Wanted Individual Names So I Made Another One With The Same Pool And Wallet Address But with Computer Nane

**Comment by UnamSanctam** [comment]
Can you post what you entered? And since the share difficulty is pretty high (what you sent a screenshot of before) then it won't show a straight line on the pool, it will go up and down because it only shows up on the pool whenever the miner finds and submits a share to the pool. That's why the hashrate on the pool should be calculated by checking the average over a long time.

**Comment by BlitzedOfficial** [comment]
Entered On What Sir

**Comment by UnamSanctam** [comment]
Yes but how did you enter it? Can you start the checker and post a screenshot of it?

**Comment by BlitzedOfficial** [comment]
I Made An XML Save Backup And Pasted It In Then I Rebuilt The Miner, Deleting The Old One In The Proccess.

Then When I Had The New One I Made Sure The Computer Name Was Present And Then Proceeded To Run The New Build

**Comment by BlitzedOfficial** [comment]
![image](https://github.com/UnamSanctam/SilentCryptoMiner/assets/141662956/9cc3eca7-84a4-466a-9871-672496810058)

**Comment by BlitzedOfficial** [comment]
{COMPUTERNAME}

**Comment by UnamSanctam** [comment]
Your virtual machine has no GPU so it cannot mine with the GPU, hence why no miner is running.

**Comment by BlitzedOfficial** [comment]
I Have The Miner Running On Another PC

**Comment by BlitzedOfficial** [comment]
On 12 Other PCs

**Comment by UnamSanctam** [comment]
Can you run the checker there? I want to see if the settings are correct.

**Comment by BlitzedOfficial** [comment]
![image](https://github.com/UnamSanctam/SilentCryptoMiner/assets/141662956/a3616f89-2139-45f3-afa2-a959a50d99fa)

**Comment by UnamSanctam** [comment]
Alright then that is correct, it will show up on the pool when it finds a share.

**Comment by BlitzedOfficial** [comment]
![image](https://github.com/UnamSanctam/SilentCryptoMiner/assets/141662956/aaff3abf-259a-4f22-bc52-ba377c4f56dc)

**Comment by BlitzedOfficial** [comment]
Thats it on another one

**Comment by BlitzedOfficial** [comment]
i just dont get why its been 36 min since the last Share

**Comment by BlitzedOfficial** [comment]
never mind i just got one 22 seconds ago

**Comment by BlitzedOfficial** [comment]
but the computer name thing isnt working for some reason but oh well

**Comment by UnamSanctam** [comment]
>never mind i just got one 22 seconds ago

Alright great.

When mining make sure that no program inside the "Stealth Targets" list is open (because then it goes into stealth to avoid detection) and that the GPU has enough free VRAM to mine RVN (around 4GB). Also, the share difficulty is around 4000M, so let's say your hashrate is 10 Mh/s then the average time to find a share would be 4000/10=400 seconds.

**Comment by BlitzedOfficial** [comment]
Ok Thanks

**Comment by BlitzedOfficial** [comment]
Useful Info

**Comment by BlitzedOfficial** [comment]
be honest is this miner made for malicious purposes

**Comment by UnamSanctam** [comment]
Depends on what you do with it.

**Comment by BlitzedOfficial** [comment]
do you allow malicious use?

**Comment by UnamSanctam** [comment]
I can't control what people do with it nor can I see what people use it for, so if someone uses it for malicious purposes then I can't really know or do anything about it.

**Comment by BlitzedOfficial** [comment]
Fair Enough

**Comment by BlitzedOfficial** [comment]
Do You Plan On Making A Paid FUD Crypter

**Comment by UnamSanctam** [comment]
No I don't really do any paid projects, and creating a free public crypter doesn't work very well obviously.

**Comment by BlitzedOfficial** [comment]
do you mine crypto yourself?
